packages/flutter/lib/src/widgets/form.dart

```diff
@@ -28,18 +28,27 @@ class Form extends StatefulWidget {
   final Widget child;
 
   @override
-  _FormState createState() => new _FormState();
+  FormState createState() => new FormState();
 }
 
-class _FormState extends State<Form> {
+class FormState extends State<Form> {
   int generation = 0;
+  Set<FormFieldState> _fields = new Set<FormFieldState>();
 
   void _onFieldChanged() {
```

Comment:
Our convention is:
* onFoo for variables that take callbacks
* handleFoo for functions that are assigned to onFoo variables
* FooCallback for the types of those variables
* didFoo for methods that you call to notify the object

So I think here you want fieldDidChange or some such.